src/main/res/values/strings.xml

```diff
@@ -139,23 +139,23 @@
   <string name="unknown_error_description">Unknown error.</string>
   <string name="unknown_format">Unknown: %s</string>
   <string name="unsigned">Unsigned</string>
-  <string name="unsigned_description">Unsigned. The list of applications could not be verified. Be careful with
-    applications downloaded from unsigned repositories.</string>
+  <string name="unsigned_description">Unsigned. Could not verify the application list. Be careful downloading
+    applications from unsigned repositories.</string>
   <string name="unstable_updates">Unstable updates</string>
-  <string name="unstable_updates_summary">Suggest updates to unstable versions</string>
+  <string name="unstable_updates_summary">Suggest installing unstable versions</string>
   <string name="unverified">Unverified</string>
   <string name="update">Update</string>
-  <string name="update_notifications">Update notifications</string>
-  <string name="update_notifications_summary">Show a notification when updates are available</string>
+  <string name="update_notifications">Notifications about newer application versions</string>
```

Comment:
(Original conversation was detached from this change)

> This line is single line at the application preferences screen. There is a switch button right to the text, so "Notifications about newer applications" will be truncated, probably. Let's keep short "Update notifications".

> Also, I already said somewhere that the client doesn't notify about new applications (which can mean, for examples, "new applications added to repository"). It notifies about new versions of already installed applications.

Also, screenshot:

![device-2020-07-06-110636](https://user-images.githubusercontent.com/24494863/86589410-3210cb00-bf7d-11ea-90d1-a4e8547a75d2.png)

Use "Notify about updates" if you don't like "update notifications".